ReactQt/tests/reacttestcase.cpp

```diff
@@ -67,6 +78,11 @@ void ReactTestCase::waitAndVerifyJsAppStarted() {
     waitAndVerifyCondition([=]() { return bridge()->jsAppStarted(); }, "js app not started, failed by timeout");
 }
 
+void ReactTestCase::waitAndVerifyJSException(const QString& exceptionMessage) {
+    waitAndVerifyCondition([=]() { return bridge()->redbox()->errorMessage() == exceptionMessage; },
+                           "Expected JS exception was not received");
```

Comment:
Adding expected exception message to output can simplify understanding the error from console.